### DISCUSSION TITLE
Language(s) 'fr de' doesn't exist in the project.

### DISCUSSION THREAD

**Discussion by beshur** [crowdin/crowdin-cli#453] [Q&A] [answered]
Hello.
I have a hard time launching a machine-translation for our project. What format should language argument should be in?
I want to launch translation for our project for all languages that's MT capable of.

`% crowdin pre-translate --method mt --engine-id 295853 --language "fr de"`
✔️  Fetching project info     
❌ Language(s) 'fr de' doesn't exist in the project. Try specifying another language code(s)

Supplying single language id as argument works fine.

Thank you.

**Comment by DimaYashchyshyn-zz** [comment]
Hi @beshur 

Try checking this article 
https://support.crowdin.com/api/language-codes/

For pre-translation, it's enough to only specify an Engine 
https://support.crowdin.com/cli-tool/#pre-translation

**Reply by beshur** [reply]
Thank you for answer, but I guess it's not enough to only specify an engine.

`% crowdin pre-translate --method mt --engine-id 295853`                  
✔️  Fetching project info     
❌ Pre-translation is running...     
❌ Failed to pre-translate the project. Please contact our support team for help
❌ Wrong parameters: 
<key: languageIds, code: notInArray, message: The input was not found in the haystack>

**Comment by DimaYashchyshyn-zz** [comment]
@beshur I see, thanks for the input. Well, in this case, as the system says, you need to contact our support team (email to support@crowdin.com) 🙂

Please include in the email project id + configuration file (without credentials, we just need to check the structure)

**Reply by beshur** [reply]
Ok, thanks

**Comment by andrii-bodnar** [comment, marked as answer]
Hi @beshur!

You need to specify the language code but in the following format:

```console
crowdin pre-translate --method mt --engine-id 295853 --language "fr"
```

Also, please try the following:

```console
crowdin pre-translate --method mt --engine-id 295853 --language "fr" --language "de"
```

`--language "fr de"` - is incorrect way

**Reply by beshur** [reply]
Thank you, I missed it.

**Comment by beshur** [comment]
Thank you, that works, as I had mentioned in the original post.
Is there a way to run pretranslation for multiple languages at once?

**Reply by andrii-bodnar** [reply]
@beshur please check my previous comment. It contains such example
